### DISCUSSION TITLE
After migrate from cloudlog to wavelog issue

### DISCUSSION THREAD

**Discussion by antonov124** [wavelog/wavelog#237] [General]
![wave](https://github.com/wavelog/wavelog/assets/27861984/41ddf35c-5b55-4e62-8bfe-10d4e509417e)
after migrate my cloud to wavelog via git ...    it looks a little bit strange,)
any ideas?

is there a wavelog support chat? @ telegram or elements or something else?

greetings DO9RAI

**Comment by HB9HIL** [comment]
'Ctrl + F5' on Windows
'Cmd + Shift + R' on Mac

Or just clear your browser cache ;)

**Comment by antonov124** [comment]
yeah ;)
thx
